### PR TITLE
change: run pennylane jobs with base container

### PR DESCRIPTION
**Pull request by virajvchaudhari**
*Issue #, if available:*

*Description of changes:*
- Adding change to run pennylane jobs with base containers
- This is also help us achieve test coverage for notebooks and containers

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by krneta** [comment]
I think someone from science should look at this. I'm not sure if there's any actual non-test reason to why we'd do the same thing twice (for different containers).

File: examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb [hunk 1]
```diff
@@ -289,13 +308,160 @@
...
+     "text": [
+      "{'params': [[0.09565119708100346, -0.0701687943257777], [-0.2417513917193508, -0.18191990185268442]], 'cost': -4.142}\n",
+      "CPU times: user 405 ms, sys: 26.7 ms, total: 432 ms\n",
+      "Wall time: 9min 46s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# This cell should take 7-8  minutes\n",
```
Comment (yitchen-tim) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb:431]: More like 10 minutes for TF

File: examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb [hunk 2]
```diff
@@ -456,27 +622,27 @@
...
+       "{'params': [[0.5304512035600284, 0.20428120976384603],\n",
+       "  [-0.31945139287717106, -0.22541990250088492]],\n",
+       " 'cost': -4.9639999999999995}"
       ]
      },
-     "execution_count": 13,
+     "execution_count": 19,
      "metadata": {},
      "output_type": "execute_result"
     }
```
Comment (yitchen-tim) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb:648]: about 9 minutes for TF containers

**Review by yitchen-tim — COMMENTED** [review]
Tested all three containers, all work for this job.

**Comment by yitchen-tim** [comment]
I don’t think there is any issue. Switching to base container for users that use only autograd makes a lot of sense. The create-to-end time reduces from 11 to about 6 min.

**Review by yitchen-tim — DISMISSED** [review]

**Review by yitchen-tim — APPROVED** [review]

**Pull request merged**